integrations/malware_tools_analyzers/Dockerfile

```diff
@@ -103,6 +103,24 @@ RUN wget -q https://github.com/mandiant/GoReSym/releases/download/v2.7.4/GoReSym
     && unzip GoReSym-linux.zip \
     && chmod +x GoReSym \
     && ln -s ${PROJECT_PATH}/goresym/GoReSym /usr/local/bin/goresym
+# Install DroidLysis
+WORKDIR "${PROJECT_PATH}/droidlysis"
+RUN python3 -m venv venv \
+ && . venv/bin/activate \
+ && mkdir -p ~/softs \
+ && cd ~/softs \
+ && wget -q https://bitbucket.org/iBotPeaches/apktool/downloads/apktool_2.9.3.jar \
+ && wget -q https://bitbucket.org/JesusFreke/smali/downloads/baksmali-2.5.2.jar \
+ && wget -q https://github.com/pxb1988/dex2jar/releases/download/v2.4/dex-tools-v2.4.zip \
+ && unzip dex-tools-v2.4.zip \
+ && rm -f dex-tools-v2.4.zip \
+ && apt-get -y install --no-install-recommends openjdk-17-jdk \
+ && pip3 install --no-cache-dir git+https://github.com/cryptax/droidlysis \
```

Comment:
please add this in a specific requirements file to have all of those dependencies ordered properly. If you put a dep here, it can't be followed by the dependabot too.

Reply:
plus, it's important to pin the version: https://pypi.org/project/droidlysis/ to have reproducibility and to follow updates